src/modules/fstab/main.py

```diff
@@ -152,20 +152,22 @@ def generate_crypttab_line_info(self, partition):
         if not mapper_name or not luks_uuid:
             return None
 
-        if mount_point == "/":
+        if mount_point != "/":
```

Comment:
Ouch, I guess unless your Debian actually wants the `/` line in crypttab which then ends up in the initramfs.